app/screens/earns-screen/earns-section.tsx

```diff
@@ -176,7 +176,7 @@ export const EarnSection: ScreenType = ({ route, navigation }: Props) => {
 
   const open = async (card) => {
     // FIXME quick fix for apollo client refactoring
```

Comment:
is this comment needed?

Reply:
We will refactor token storage in the mobile app and probably use apollo client. I think (?) this comment is referring to that. There are a lot of fixmes (26 in 18 files) throughout the mobile codebase, and I don't want to remove them before addressing them.